app/jobs/suitable_for_early_allocation_email_job.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class SuitableForEarlyAllocationEmailJob < ApplicationJob
+  queue_as :mailer
+
+  EQUIP_URL = 'https://equip-portal.rocstac.com/CtrlWebIsapi.dll/?__id=webDiagram.show&map=0%3A9A63E167DE4B400EA07F81A9271E1944&dgm=4F984B45CBC447B1A304B2FFECABB777'
+
+  def perform
+    offenders = EarlyAllocation.where(created_within_referral_window: false).where.not(outcome: 'ineligible').pluck(:nomis_offender_id).uniq
+
+    offenders.each do |offender_no|
+      allocation = Allocation.where(nomis_offender_id: offender_no).first
+      next if allocation.nil?
+
+      prisoner = OffenderService.get_offender(offender_no)
+      if !prisoner.nil? && eligible_today?(prisoner)
```

Comment:
I worry about these edge-valued tests - if we miss a day for any reason, then the test will be false and the mail won't get sent. We have an EmailHistory model now which maybe we should be using here? - wonder if we could use it to make the test threshold-based instead?

Reply:
Hopefully the changes I've made fixes this